daml-lf/verification/stainless.nix

```diff
@@ -0,0 +1,17 @@
+let
+  pkgs = import <nixpkgs> {};
+  stdenv = pkgs.stdenv;
+in rec {
+  stainlessEnv = stdenv.mkDerivation rec {
+    name = "stainless-env";
+    shellHook = ''
+    alias cls=clear
+    '';
+    buildInputs = with pkgs; [
+      stdenv
+      sbt
+      openjdk
```

Comment:
Can we specify somehow that the jdk must be >=v17?

Reply:
I've specified JDK 17